bot/bot.go

```diff
@@ -572,6 +577,15 @@ func (b *Bot) dealWithHTTPTimeouts(event *slack.MessageEvent) {
 	}
 }
 
+func (b *Bot) sliceUsageAndInternals(event *slack.MessageEvent) {
+	params := slack.PostMessageParameters{AsUser: true}
+	_, _, err := b.slackBotAPI.PostMessage(event.Channel, `<https://blog.golang.org/go-slices-usage-and-internals/>`, params)
```

Comment:
I see other commands sending back links in the same odd way with a trailing slash like it's closing a tag. Looks like I assumed too much. Good catch.